src/main/resources/db/migration/V1_5__create_courthouse_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE moj_courthouse (
+  moj_crt_id                INTEGER                         GENERATED ALWAYS AS IDENTITY
+, courthouse_name           CHARACTER VARYING               NOT NULL
+, code                      SMALLINT
```

Comment:
I can see that Barry's v18 sql has
`courthouse_code           CHARACTER VARYING         NOT NULL`

I'm thinking that is what we want to have here? It might be a mistake to be numeric

Reply:
i think it must be a mistake in Barry's.

Reply:
Barry has confirmed that this is a incorrect and should be an integer.